packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -44,57 +44,204 @@ const isDropdownSelect = (element = {}) => {
 
 export default class Calendar extends React.Component {
   static propTypes = {
+    /**
+     * Prop to change date when a valid date is selected.
+     */
     adjustDateOnChange: PropTypes.bool,
+    /**
+     * Class name to style the date picker.
+     */
     className: PropTypes.string,
+    /**
+     * Component to render within date picker.
+     */
     children: PropTypes.node,
+    /**
+     * Format of the date selected.
+     */
     dateFormat: PropTypes.oneOfType([
       PropTypes.string,
       PropTypes.array
     ]).isRequired,
+    /**
+     * Prop to style individual days on calendar.
+     */
     dayClassName: PropTypes.func,
+    /**
+     * Prop to select a dropdown mode for the year dropdown.
+     */
     dropdownMode: PropTypes.oneOf(['scroll', 'select']).isRequired,
+    /**
+     * Maximum Date for a given range.
+     */
     endDate: PropTypes.object,
+    /**
+     * Array to exclude certain dates.
+     */
     excludeDates: PropTypes.array,
+    /**
+     * A callback function to be executed when filter dates is triggered.
+     */
     filterDate: PropTypes.func,
+    /**
+     * Prop Keep the height of calendar dom fixed or variable
```

Comment:
Can be rephrased to be more clear. Something like "Specifies whether the height of the calendar is fixed or variable"